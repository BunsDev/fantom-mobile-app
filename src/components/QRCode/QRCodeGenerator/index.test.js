import React from 'react';
import { shallow } from 'enzyme';

import Tab from './index';

test('renders correctly', () => {
  const wrapper = shallow(<Tab />);
  expect(wrapper).toMatchSnapshot();
});
