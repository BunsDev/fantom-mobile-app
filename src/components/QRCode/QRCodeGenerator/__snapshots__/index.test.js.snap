// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "rgb(44,52,58)",
        "borderRadius": 4,
        "justifyContent": "center",
        "marginBottom": 24,
        "padding": 12,
        "width": 718,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#FFF",
          "fontFamily": "SFProDisplay-Semibold",
          "fontSize": 16,
        }
      }
    >
       
       
    </Text>
  </View>
  <View
    style={
      Object {
        "alignSelf": "center",
        "flexDirection": "row",
        "height": 350,
      }
    }
  >
    <ActivityIndicator
      animating={true}
      color="#FFF"
      hidesWhenStopped={true}
      size="large"
    />
  </View>
</View>
`;
